docs/Release.md

```diff
@@ -43,6 +43,10 @@ This table describes all officially supported MsQuic releases.
 <br>\* Future **Release Dates** are subject to change.
 <br>\** **End of Support** dates do not include possible [extended support](https://docs.microsoft.com/en-us/windows-server/get-started-19/servicing-channels-19#long-term-servicing-channel-ltsc) extensions.
 
+## Architectures
+
+Generally, MsQuic is supported for all 64-bit architectures on all platforms, including x64 and arm64. arm32 is also supported, but only on Linux.
```

Comment:
I personally think we should have a specific list (and potentially a "while not officially supported, MsQuic should also work on any 64-bit architecture and arm32 on linux).
"Generally" is not that helpful to me for my exotic architecture / platform, or could be seen as a commitment to support something we actually don't intend to, even if we try our best to be compatible with everything.